pkg/solana/logpoller/log_poller.go

```diff
@@ -27,43 +28,80 @@ type ORM interface {
 	DeleteFilters(ctx context.Context, filters map[int64]Filter) error
 	MarkFilterDeleted(ctx context.Context, id int64) (err error)
 	MarkFilterBackfilled(ctx context.Context, id int64) (err error)
+	GetLatestBlock(ctx context.Context) (int64, error)
```

Comment:
Why use the ORM to give the latest block? Weren't we getting that from the RPC?

Reply:
LogPoller needs to provide a guarantee that it captures all events. If we only get the latest block from RPC, we'll miss some due to restarts or other downtime.

Reply:
@EasterTheBunny These are two different things--we need to know what block we left off at before a restart, as well as what the latest block on chain is.  The naming can be confusing since "GetLatestBlock" doesn't indicate which, but I think it's okay because it's a part of the ORM so it's clear which one you're requesting. (Use the ORM to get latest in db, use rpc server to get latest from chain. Then backfill everything in between.)